### ISSUE TITLE
CTJ Onboarding Questions and Comments [Peter - UI/UX Designer]

### ISSUE THREAD

**Issue by kcoronel** [hackforla/CivicTechJobs#546]
### Overview
This issue has been created for our new UI/UX designer to give us comments and suggestions related to our current project onboarding project.

### Action Items
 While reviewing the documents listed below, please document open questions and suggestions as comments in this issue

- [x] Review [Read Me Page](https://github.com/hackforla/CivicTechJobs)
- [x] Review [Wiki Page](https://github.com/hackforla/CivicTechJobs/wiki)
- [x] Review [Mk Docs Project Documentation](https://hackforla.github.io/CivicTechJobs/)
- [ ] Review [Kanban Board Docs and Onboarding column](https://github.com/hackforla/CivicTechJobs/projects/1)

**Comment by TimiPeters** [comment]
During my onboarding, I found myself clicking through multiple links to access the necessary documents and information. It felt a bit scattered, and I think it would be much easier if everything was available in one place, so people don’t have to jump around. Also, I noticed that the final link didn’t work, which added some frustration. Making these changes could really smooth out the onboarding experience.
